bin/install.sh

```diff
@@ -17,47 +17,72 @@
 # along with this program; if not, write to the Free Software
 # Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
 
+# Check if script is being run as root
+if (( $EUID == 0 )); then
+    read -p "For most use cases, running this script as root is NOT recommended. Are you sure? Y/N" yesno
 
-# Add the terraform repository from HashiCorp
-# for reference: https://developer.hashicorp.com/terraform/cli/install/yum
-# Currently supported distros: fedora, RHEL
+    case $yesno in
+        [Yy]* )
+            echo "You answered yes, continuing install as root." ;;
+        [Nn]* )
+            echo "You answered no, exiting"; exit 1 ;; 
+        *) 
+            echo "Unknown input, exiting"; exit 1 ;;
+    esac
+fi
 
-# Get the operating system distribution
-os_release=$(grep "^ID=" /etc/os-release | awk -F'=' '{print $2}')
+if (( $EUID != 0 )); then
```

Comment:
change this to an else.  If EUID check for 0 is above, so else is automatically everything else.